src/test/java/seedu/weme/storage/JsonMemeBookStorageTest.java

```diff
@@ -55,11 +55,6 @@ public void readMemeBook_invalidMemeMemeBook_throwDataConversionException() {
         assertThrows(DataConversionException.class, () -> readMemeBook("invalidMemeMemeBook.json"));
     }
 
-    @Test
-    public void readMemeBook_invalidAndValidMemeMemeBook_throwDataConversionException() {
```

Comment:
keep this test, make the invalid entry in the json one with an invalid name instead.